docs/index.rst

```diff
@@ -3411,18 +3411,69 @@ The only required argument to create an ensemble is the dataset id:
 
     ensemble = api.create_ensemble('dataset/5143a51a37203f2cf7000972')
 
-but you can also specify the number of models to be built and the
-parallelism level for the task:
+BigML offers three kinds of ensembles. Two of them are known as ``Decision
+Forests`` because they are built as collections of ``Decision trees``
+whose predictions
+are aggregated using different combiners: ``plurality``,
+``confidence weighted``, ``probability weighted`` or setting a ``threshold``
+to issue the ensemble's
+prediction. All ``Decision Forests`` use bagging to sample the
+data used to build the underlying models.
+
+As an example of how to create a ``Decision Forest``
+with `20` models, you only need to provide the dataset ID that you want to
+build the ensemble from and the number of models:
+
+.. code-block:: python
+
+    args = {'number_of_models': 20}
+    ensemble = api.create_ensemble('dataset/5143a51a37203f2cf7000972', args)
+
+If no ``number_of_models`` is provided, the ensemble will contain 10 models.
+
+``Random Decision Forests`` fall
+also into the ``Decision Forest`` category,
+but they also use a subset of the fields chosen
+at random at each split. To create this kind of ensemble, just use the
+``randomize`` option:
 
 .. code-block:: python
 
-    args = {'number_of_models': 20, 'tlp': 3}
+    args = {'number_of_models': 20, 'randomize': True}
+    ensemble = api.create_ensemble('dataset/5143a51a37203f2cf7000972', args)
+
+The third kind of ensemble is ``Boosted Trees``. This type of ensemble uses
+quite a different algorithm. The trees used in the ensemble don't have as
+objective field the one you want to predict, and they don't aggregate the
+underlying models' votes. Instead, the goal is adjusting the coefficients
+in a function, which will be used to predict. The
+models' objective is, therefore, the gradient
+that you need to improve the error
+between the real values and the function prediction. The process starts with
+some initial values and compute these gradients. Next step uses the previous
+fields plus the last computed gradient field as
+the new initial state for the next iteration.
+Finally, it stops when predictions
+match the
+expected results or iterations reach a user-defined limit.
```

Comment:
i guess it's not an exact match but rather when prediction error is below a given threshold?